jax/_src/config.py

```diff
@@ -1106,6 +1106,15 @@ def _validate_jax_pjrt_client_create_options(new_val):
     help=('Use direct linearization instead JVP followed by partial eval'),
     include_in_jit_key=True)
 
+use_simplified_jaxpr_constants = bool_state(
```

Comment:
nit: I think you need to add `include_in_jit_key=True` here and also add it to `trace_context` above.

Reply:
done